### PR TITLE
Add basic Django assets

### LINKED ISSUE #5: Add basic Django assets
## Description
Currently the master branch is hosting the Github Pages + Jekyll powered landing page. We should therefore work on `develop` branch to create the web app that will be TAKWIMU.

## Tasks

- [x] Create a feature branch with the basic workings of a Django app
- [x] Deploy app to staging server
- [ ] ?

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Currently the master branch is hosting the Github Pages + Jekyll powered landing page. We should therefore work on develop branch to create the web app that will be TAKWIMU.

## Tasks

- [x] Create a feature branch with the basic workings of a Django app
- [x] Deploy app to staging server. App deployed to http://takwimu.dokku-1.hurumap.org

Fixes #5 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Screenshots
screencapture-localhost-8000-2018-04-24-11_18_32

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by phillipahereza** [comment]
@DavidLemayian changes made

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: gunicorn TAKWIMU.wsgi
```
Comment (DavidLemayian) [inline Procfile]: @phillipahereza This might need a change?

File: runtime.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+python-2.7.14
```
Comment (DavidLemayian) [inline runtime.txt]: @phillipahereza Do you think 3.6 might offer performance improvements?

File: takwimu/settings.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+import dj_database_url
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'a526!hvj-npei9ex35)+venm$yd%*gr+ri+uh59sfrstbt#60k'
```
Comment (DavidLemayian) [inline takwimu/settings.py]: This should be set using ENV variable. We can have a default though for dev purposes.

File: runtime.txt (hunk 2 above)
Comment (phillipahereza) [reply runtime.txt]: @DavidLemayian but HURUmap runs on 2.7.14, wouldn't it be a safe bet to use the same version?

Comment (DavidLemayian) [reply runtime.txt]: @phillipahereza Safe bets are the enemy of progress. ;)

Comment (phillipahereza) [reply runtime.txt]: @DavidLemayian in that case, should we also checkout django 2.0?

File: templates/_base.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+{% block main_content %}
+
+{% endblock %}
+
+<footer class="text-center pb-4">
+			<p class="small">All content released under a <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noopener">Creative Commons 4 Attribution license</a>.</p>
+		</footer>
+
+</body>
+</html>
```
Comment (DavidLemayian) [inline templates/_base.html]: Please fix indentation.

**Pull request merged**